bin/rucio-admin

```diff
@@ -12,6 +12,7 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+
```

Comment:
No argument against what you’re trying to achieve here (pluse the use of double quotes), but please explain in your commit message the _why_ (e.g. referencing PEP 257).

Reply:
Addressed in https://github.com/rucio/rucio/pull/7472/commits/9c326130168f95c935ffe4b4d8025e32880e1ff4